.gitignore

```diff
@@ -0,0 +1,3 @@
+/vendor
+/.idea
```

Comment:
NO. This should not go i here. This belongs in your global .gitignore file. People always do this wrong. :(